### PR TITLE
Update mnist_poison_detection.py

### PR DESCRIPTION
**Pull request by lea4n**
Fix bug

# Description

Fix the bug when init Class 'KerasClassifier', from "classifier = KerasClassifier((min_, max_), model=model)" to "classifier = KerasClassifier(model=model)", because during init KerasClassifier, it isn't need three parameters.

Fixes # (issue)
mnist_poison_detection.py example error #133

## Type of change

Please check all relevant options.

[ ] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing

Please describe the tests that you ran to verify your changes. Consider listing any relevant details of your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
- OS: Ubuntu 16.04.1 LTS and kernel is 4.4.0-57-generic
- Python version: 3.7.3(Anaconda)
- ART version or commit number: master
- TensorFlow / Keras / PyTorch / MXNet version: Keras,2.2.4 and TensorFlow,1.14.0

# Checklist

[ ] My code follows the style guidelines of this project
[ ] I have performed a self-review of my own code
[ ] I have commented my code
[ ] I have made corresponding changes to the documentation
[ ] My changes generate no new warnings
[ ] I have added tests that prove my fix is effective or that my feature works
[ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by beat-buesser** [comment]
Hi @lea4n Thank you for the PR!

File: examples/mnist_poison_detection.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def main():
 
     model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])
 
-    classifier = KerasClassifier((min_, max_), model=model)
+    classifier = KerasClassifier(model=model)
```
Comment (beat-buesser) [inline examples/mnist_poison_detection.py]: `clip_values` is now a keyword argument. We'd like to keep the clip_values in the example to demonstrate its use. How about this:
```suggestion
    classifier = KerasClassifier(model=model, clip_values=(min_, max_))
```

Comment (lea4n) [reply examples/mnist_poison_detection.py]: fine

**Comment by beat-buesser** [comment]
@lea4n Thank you for your first contribution to ART!

**Pull request merged**